### DISCUSSION TITLE
deploying a Nodejs app with serverless function error

### DISCUSSION THREAD

**Discussion by ahmedelazab844** [vercel/community#2249] [Help]
### Summary

Need help to figure what is the wrong with my deployments, the node app is serving the frontend content after several refreshes, i have a huge frontend content so maybe the hobby is not helping to serve the app properly, but the core problem is the application apis are not working at all

**Comment by amyegan** [comment]
Hi @ahmedelazab844! You should be able to get more error information from the [Runtime Logs](https://vercel.com/docs/concepts/observability/runtime-logs#). If you can share your app or a minimal reproduction here, someone may be able to recognize the source of the problem by getting a closer look. Short of that, sharing the error details and additional information about your project setup may help us point you in the right direction
